treeherder/perf/fixtures/performance_bug_templates.yaml

```diff
@@ -14,6 +14,8 @@
 
       If you need the profiling jobs you can trigger them yourself from treeherder job view or ask a sheriff to do that for you.
 
+      You can run these tests on try with ./mach try perf --alert <SUMMARY-ID>
```

Comment:
Awesome @junngo, thanks for looking into this!

One question I have is could we add a new template argument here so that we could provide the summary id directly in the command? With that we could change the <SUMMARY-ID> to {{ summaryId }} and developers would be able to copy it to trigger the CI tasks: https://github.com/mozilla/treeherder/blob/0512e42f73603acccd7332f0901aaa11c327106b/ui/perfherder/alerts/StatusDropdown.jsx#L112-L119

I also think you can use the back-ticks " ` " here to make it appear like code in bugzilla.

Reply:
Thanks for reviewing :)
That is a good idea👍 
It looks okay If we use the `alertSummary.id`. Also, I'll add the back-ticks to emphasize the command.